docker/openjdk-11/Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-FROM openjdk:11-jre-slim-buster
+FROM eclipse-temurin:11-jre
```

Comment:
(a) The variant for this docker file is, explicitly by tag and directory name, `openjdk-11`. If the solutionis to stop using openjdk-11 the the tag and directory name should be changed to `eclipse-temurin-11`

(b) There are two other packages using `openjdk:11-jre-slim-buster` in this repo, any patch should also address those two docker images